sdk/logical/request.go

```diff
@@ -203,7 +203,7 @@ type Request struct {
 
 	// ResponseWriter if set can be used to stream a response value to the http
 	// request that generated this logical.Request object.
-	ResponseWriter *HTTPResponseWriter `json:"-" sentinel:""`
+	ResponseWriter *http.ResponseWriter `json:"-" sentinel:""`
```

Comment:
I'd rather use the explicit type here than the interface, since there aren't any cases where we'll be using anything other than that type.